nextcord/flags.py

```diff
@@ -884,6 +912,8 @@ def scheduled_events(self):
         - :func:`on_guild_scheduled_event_create`
         - :func:`on_guild_scheduled_event_delete`
         - :func:`on_guild_scheduled_event_update`
+        - :func:`on_guild_scheduled_event_user_add`
```

Comment:
Unrelated changes?

Reply:
It's not related to this change, but I saw it in the [docs](https://github.com/discord/discord-api-docs/blob/3891389606ef8710e7c54d65ae79d0c5313f0247/docs/topics/Gateway.md) when making this change.

Do you want theses lines to be reverted?

Reply:
Yeah that wouls be awesome!